poly-commit/src/linear_codes/ligero.rs

```diff
@@ -96,6 +97,23 @@ where
     }
 }
 
+impl<F, C, H> Absorb for LigeroPCParams<F, C, H>
+where
+    F: PrimeField,
+    C: Config,
+    H: CRHScheme,
+{
+    fn to_sponge_bytes(&self, dest: &mut Vec<u8>) {
```

Comment:
You need to import `ark_std::vec::Vec` under the `no_std` feature flag. If you want to test the library with `no_std` you can install `cross` (`cargo install cross`) and then:
```
cargo cross build --workspace --no-default-features --target aarch64-unknown-none --exclude ark-pcs-bench-templates
```